lib/routes/routeBackbeat.js

```diff
@@ -141,6 +146,26 @@ function _checkMultipleBackendRequest(request, log) {
     return undefined;
 }
 
+function getPartList(parts, objectKey, uploadId, storageLocation) {
+    const partList = {};
+    if (locationConstraints[storageLocation].type === 'azure') {
+        partList.uncommittedBlocks = [];
```

Comment:
You can use map to make this cleaner.
```js
partList.uncommittedBlocks = parts.map(part => {


```

Reply:
I want to use spread syntax here to push all items of the array `subPartIds`.

https://github.com/scality/S3/pull/1054/files/6547c168ac5060473d1da7904e86fece26c664e5#diff-5c3dc4783e2d0f782449eea237c8f691R161